ui/templates/base.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <script type="text/javascript" src="{{ style_src }}"></script>
+    {% load staticfiles %}
+    <link href="{% static 'css/layout.css'%}" rel="Stylesheet"/>
```

Comment:
After the contents of `layout.css` are copied into `layout.scss` you should remove this line. The contents of `layout.scss` are included in the above line with `style_src` instead.